docs/src/js/ds4/main.js

```diff
@@ -7,8 +7,36 @@ function querySelectorAllToArray(selector, parentNode) {
     return nodeListToArray(parentNode.querySelectorAll(selector));
 }
 
-$('.tooltip').tooltip();
-$('.bubblehelp').clickFlyout({triggerSelector: '.bubblehelp__host', overlaySelector: '.bubblehelp__overlay', autoCollapse: false});
+// TOOLTIP WIDGETS - Expand/Collapse only
+querySelectorAllToArray('.tooltip').forEach(function(el, index) {
```

Comment:
Please don't pay too much attention to the following js. It's not going to be around for very long! Unless there are any glaring errors of course!